diff --git a/node_modules/next-seo/lib/index.d.ts b/node_modules/next-seo/lib/index.d.ts
index 4b5a1ae..ea4995c 100644
--- a/node_modules/next-seo/lib/index.d.ts
+++ b/node_modules/next-seo/lib/index.d.ts
@@ -23,4 +23,4 @@ export { default as SiteLinksSearchBoxJsonLd, SiteLinksSearchBoxJsonLdProps, } f
 export { default as SoftwareAppJsonLd, SoftwareAppJsonLdProps, } from './jsonld/softwareApp';
 export { default as CollectionPageJsonLd, CollectionPageJsonLdProps, } from './jsonld/collectionPage';
 export { default as ProfilePageJsonLd, ProfilePageJsonLdProps, } from './jsonld/profilePage';
-export { DefaultSeoProps, NextSeoProps } from './types';
+export * from './types';
